runtime/doc/faq.txt

```diff
@@ -0,0 +1,701 @@
+passed through to Nvim.
+
+
+CALLING INPUTLIST(), ECHOMSG, ... IN FILETYPE PLUGINS AND AUTOCMD DOES NOT WORK ~
+
+#10008 https://github.com/neovim/neovim/issues/10008,
+#10116 https://github.com/neovim/neovim/issues/10116,
+#12288 https://github.com/neovim/neovim/issues/12288,
+# vim/vim#4379 https://github.com/vim/vim/issues/4379.
+This is because Nvim sets `shortmess+=F` by default. Vim behaves the same way
+with `set shortmes+=F`. There are plans to improve this, but meanwhile as a
+workaround, use `set shortmess-=F` or use `unsilent` as follows.
+>vim
+    unsilent let var = inputlist(['1. item1', '2. item2'])
+    autocmd BufNewFile * unsilent echomsg 'The autocmd has been fired.'
+<
+
+G:CLIPBOARD SETTINGS ARE NOT USED. ~
+
+If the clipboard provider is already loaded, you will need to reload it after
+configuration. Use the following configuration.
+>vim
+    let g:clipboard = { 'name' : ... }
+    if exists('g:loaded_clipboard_provider')
+      unlet g:loaded_clipboard_provider
+      runtime autoload/provider/clipboard.vim
+    endif
+<
+
+Or, if you want automatic reloading when assigning to |g:clipboard|, set
+|init.vim| as follows.
+>vim
+    function! s:clipboard_changed(...) abort
+      if exists('g:loaded_clipboard_provider')
+        unlet g:loaded_clipboard_provider
+      endif
+      runtime autoload/provider/clipboard.vim
+    endfunction
+
+    if !exists('s:loaded")
+      call dictwatcheradd(g:, 'clipboard', function('s:clipboard_changed'))
+    endif
+    let s:loaded = v:true
+<
+
+==============================================================================
+3. Installation issues                               *faq-installation-issues*
+
+
+GENERATING HELPTAGS FAILED ~
+
+If re-installation fails with "Generating helptags failed", try removing the
+previously installed runtime directory (if `CMAKE_INSTALL_PREFIX` is not set
+during building, the default is `/usr/local/share/nvim`):
+>bash
+    rm -r /usr/local/share/nvim
+<
+
+==============================================================================
+4. Build issues                                             *faq-build-issues*
+
+
+GENERAL BUILD ISSUES ~
+
+Run `make distclean && make` to rule out a stale build environment causing the
+failure.
+
+
+SETTINGS IN LOCAL.MK DON'T TAKE EFFECT ~
+
+CMake caches build settings, so you might need to run `rm -r build && make`
+after modifying `local.mk`.
+
+
+CMAKE ERRORS ~
+
+`configure_file Problem configuring file`
+
+This is probably a permissions issue, which can happen if you run `make` as the
+root user, then later run an unprivileged `make`. To fix this, run `rm -rf
+build` and try again.
+
+
+ANACONDA ERROR ~
+
+Error message: `anaconda3/bin/x86_64-conda_cos6-linux-gnu-cc: not found`
+
+Solution: `conda install gxx_linux-64` or `conda deactivate`
+
+
+==============================================================================
+5. Debug                                                           *faq-debug*
```

Comment:
That whole section probably belongs in CONTRIBUTING.md or MAINTAINING.md (or a new `:h dev_tools` file or somesuch); it's a bit out of place here.

Reply:
`dev_tools.txt` sounds good